build/configs/rtl8720e/hello/defconfig

```diff
@@ -24,7 +24,6 @@ CONFIG_BUILD_FLAT=y
 # CONFIG_APP_BINARY_SEPARATION is not set
 # CONFIG_BUILD_2PASS is not set
 CONFIG_BOARD_BUILD_DATE="200204"
-# CONFIG_BINARY_SIGNING is not set
```

Comment:
Could you revert this change?

Reply:
The image signing mechanism is not available in TP1e TizenRT master, we have provided a signing script.
All image signing and encryption can be done via script.

Reply:
@ziyik It's the same as TP1x. This config is necessary to decide whether your script should be executed or not internally.
Because it's impossible, we disable it by default. And no way to enable and even if it's enabled, nothing will be called.
Could you keep it as disabled? (revert the change)

Reply:
We will keep the config but it does nothing even if it's been called, is this the correct understanding?

Reply:
correct.